hypha/apply/funds/forms.py

```diff
@@ -327,13 +336,25 @@ def __init__(self, *args, user=None, round=None, **kwargs):
             self.fields[field_name] = data['field']
             self.role_fields[field_name] = data['role']
 
+        self.fields.move_to_end('reviewer_reviewers')
+
     def clean_submissions(self):
         value = self.cleaned_data['submissions']
         submission_ids = [int(submission) for submission in value.split(',')]
         return ApplicationSubmission.objects.filter(id__in=submission_ids)
 
     def clean(self):
         cleaned_data = super().clean()
+        external_reviewers = self.cleaned_data['reviewer_reviewers']
+        submissions = self.cleaned_data['submissions']
+        if external_reviewers:
+            # User needs to be superuser or lead of all selected submissions.
+            if self.user_cant_alter_submissions_external_reviewers(submissions, self.user):
+                self.add_error('reviewer_reviewers', _("Only Lead can change the External Reviewers"))
```

Comment:
Would it be possible to have the error messages say "External Reviewer" instead of "reviewer_reviewers"?

Reply:
Actually, 'reviewer_reviewers' is the field name defined [here in form](https://github.com/HyphaApp/hypha/pull/2831/files#diff-8c72fbe57208c4b39ad8032b4678b32ad493b198a736813deae5efaa79b26797R318) and it is coming from there.  I tried to make it similar with existing [UpdateReviewerForm field](https://github.com/HyphaApp/hypha/blob/main/hypha/apply/funds/forms.py#L193). We may change 'reviewer_reviewers' to 'external_reviewers', but i don't think it can be in capitalized form. 
I also tried by passing non-field error but in that case it was showing `__all__` in place of 'reviewer_reviewers'.